packages/js/customer-effort-score/src/components/customer-effort-score/index.tsx

```diff
@@ -54,11 +63,14 @@ type CustomerEffortScoreProps = {
  * @param {Function} props.onModalDismissedCallback  Function to call when modal is dismissed.
  * @param {Function} props.shouldShowComments        Callback to determine if comments section should be shown.
  * @param {Object}   props.icon                      Icon (React component) to be shown on the notice.
+ * @param {Object}   props.extraFields               Function that returns the extra fields to be shown.
```

Comment:
Something seems wrong here. I think it's a Function, not an Object. If this is a function, I would rename it, because from this name, I can't tell what it's doing. Based on the description, maybe `getExtraFieldsToBeShown`?

Reply:
Good idea!